docs/src/docs/A11yInterface/examples/OneDataset.js.example

```diff
@@ -0,0 +1,24 @@
+const A11yInterfaceExample = (props) => {
+  const data = _.range(10).map(d => ({
+    x: d,
+    y: Math.round(d * Math.random() * 10)
+  }))
+  return <XYPlot scaleType="linear" width={600} height={350}>
+      <XAxis title="X Axis" />
+      <YAxis title="Y Axis" />
+      <LineChart
+        data={data}
+        x={d => d.x}
+        y={d => d.y}
+        lineStyle={{stroke: '#ff7f0e', strokeWidth: 3}}
+      />
+      <A11yInterface
+        ariaLabelGenerator={frameIndex => 
+        `x Value: ${data[frameIndex].x}, y Value: ${data[frameIndex].y}`
+        }
+        numFrames={data.length}
+       />
```

Comment:
I had this idea to simplify the API by using dependency injection and injecting the `A11yInterface` into `LineChart` so that a) the API is more declarative and b) we can be more explicit about which components support keyboard navigation. Then I realized it doesn't work for multiple line charts in the same XYPlot which is probably why you didn't go with this in the 1st place!

Still sharing for discussion. Is there any way we could do something like this and still support multiple charts in the same XYPlot?

```jsx
return <XYPlot scaleType="linear" width={600} height={350}>
      <XAxis title="X Axis" />
      <YAxis title="Y Axis" />
      <LineChart
        data={data}
        x={d => d.x}
        y={d => d.y}
        lineStyle={{stroke: '#ff7f0e', strokeWidth: 3}}
        ariaLabel={(d, index_in_case_we_need_it) =>  {/* rename ariaLabelGenerator -> ariaLabel */}
        `x Value: ${d.x}, y Value: ${d.y}`
        }
        numAriaLabels={data.length} {/* Do we even need this? Could be optional and default to data.length*/}
      />
    </XYPlot>;
```

I'm guessing inside of `LineChart` we'd add something like this:

```jsx
{ props.ariaLabel ?
<A11yInterface
  ariaLabelGenerator={props.ariaLabel}
  numFrames={props.numAriaLabels || props.data.length}
/> : null
}
```

We can even provide a sensible default `ariaLabelGenerator` function for LineChart and then the caller could get this for free.

Reply:
As I mentioned in another comment, I'd prefer to leave this as a standalone component because this would break if we were to use multiple LineCharts, or Line + Area Charts. My project leverages rendering multiple data in different linecharts, and this built in schema would not work for that use case. However if we export the `A11yInterface` as an optional component and do this as well, I think that would be great.